freqtrade/persistence/models.py

```diff
@@ -703,6 +705,7 @@ class Trade(_DECL_BASE, LocalTrade):
     sell_reason = Column(String(100), nullable=True)
     sell_order_status = Column(String(100), nullable=True)
     strategy = Column(String(100), nullable=True)
+    buy_signal_name = Column(String(100), nullable=True)
```

Comment:
When adding columns, you'll need to
* add them to the relevant sections in `migrations.py`
* Make sure the last column addition is in `check_migrate()` in `migrations.py` (so the migration knows if a migration is necessary